composer.json

```diff
@@ -97,8 +97,8 @@
         "webmozart/assert": "^1.1",
         "white-october/pagerfanta-bundle": "^1.0.8",
         "willdurand/hateoas": "^2.12",
-        "willdurand/hateoas-bundle": "^1.2",
-        "winzou/state-machine-bundle": "^0.3",
+        "willdurand/hateoas-bundle": "^1.2 || ^2.0",
+        "winzou/state-machine-bundle": "^0.3 || ^4.0",
```

Comment:
I guess this change can be reverted, there's no `winzou/state-machine-bundle` in `^4.0`.

Reply:
I removed the last commit from your PR, so we can get it merged faster 🎉